src/store/ViewStore.js

```diff
@@ -78,6 +101,9 @@ export default class ViewStore {
     }
 
     inputRef = (ref) => {
-        if (ref) { ref.focus(); }
+        if (ref) {
+            ref.value = this.templateName;
+            ref.focus();
```

Comment:
if `focus` is only called during rerenders, you mind as well use `autoFocus` and eliminate needing to use `ref`.